modules/developer-cli-odo-installing-odo-on-linux.adoc

```diff
@@ -6,88 +6,40 @@
 
 = Installing {odo-title} on Linux
 
-== Binary installation
+The `{odo-title}` CLI is available to download as a binary and as a tarball for multiple operating systems and architectures including: 
 
-.Procedure
-
-. Obtain the binary:
-+
-[source,terminal]
-----
-# curl -L https://mirror.openshift.com/pub/openshift-v4/clients/odo/latest/odo-linux-amd64 -o /usr/local/bin/odo
-----
-
-. Change the permissions on the file:
-+
-[source,terminal]
-----
-# chmod +x /usr/local/bin/odo
-----
+[cols="2,1,1",options="header"]
+|===
+|Operating System|Binary|Tarball
+|Linux|link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/odo-linux-amd64[odo-linux-amd64] |link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/odo-linux-amd64.tar.gz[odo-linux-amd64.tar.gz]
+|Linux on IBM Power|link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/odo-linux-ppc64le[odo-linux-ppc64le] |link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/odo-linux-ppc64le.tar.gz[odo-linux-ppc64le.tar.gz]
+|Linux on IBM Z and LinuxONE|link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/odo-linux-s390x[odo-linux-s390x] |link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/odo-linux-s390x.tar.gz[odo-linux-s390x.tar.gz]
+|===
 
-== Tarball installation
 
 .Procedure
 
-. Obtain the tarball:
+. Navigate to link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/[] and download the appropriate file for your operating system and architecture.
```

Comment:
Give the link a name. Something like, 

Navigate to the page, `link:<link>[odo latest]`, and download the appropriate file for your operating system and architecture.

Reply:
Agreed. I have reworked the section to add table to downloads, clarify binary vs tarball differences